app/main.py

```diff
@@ -5,22 +5,32 @@ def __init__(
             clean_mark: int = 1,
             brand: str = "some_car"
     ) -> None:
-        self.comfort_class = comfort_class
-        self.clean_mark = clean_mark
+        if comfort_class in range(1, 7 + 1):
+            self.comfort_class = comfort_class
+
+        if clean_mark in range(1, 10 + 1):
+            self.clean_mark = clean_mark
```

Comment:
what if I pass -100?
it means you'll not set self.comfort_class at all and the same for other attributes
basically you can still use if statement or even create a separate function, but if statement should be correct and value should be set as well :)